src/Administration/Resources/app/administration/src/app/component/structure/sw-search-bar/sw-search-bar.scss

```
@@ -64,15 +62,16 @@ $sw-search-bar-types-item-border-radius: $border-radius-default;
         margin-bottom: 0;
         border: 1px solid $sw-search-bar-border-color;
         border-radius: $sw-search-bar-border-radius;
-        background-color: $color-white;
+        background-color: var(--color-background-primary-default);
```

Should we change this to:
```suggestion
        background-color: var(--color-elevation-surface-raised);
```
to be compatible with the current tokens version?